src/legacy/core_plugins/input_control_vis/public/components/vis/form_row.js

```diff
@@ -47,7 +47,7 @@ export function FormRow(props) {
 }
 
 FormRow.propTypes = {
-  label: PropTypes.string.isRequired,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
```

Comment:
Instead of passing in a node for label that includes the tooltip, How about passing in `warningMsg` prop to `FormRow` and then have FormRow render the icon and tooltip?

Reply:
Makes sense, I moved the logic down to `FormRow`